### PR TITLE
Fixes the message feedback when you place someone into a shelter, and adds click dragging

### PR DESCRIPTION
**Pull request by ProfanedBane**
visible_message wasn't used correctly it seems meaning you got no feedback when placing someone else into it, confusing players.

Also adds click dragging yourself and others into the shelter, copy pasted from cryo code so reviewers please have a look incase I bunged it up. Nothing appeared to go wrong in tests.

Pretty sure this won't conflict https://github.com/vgstation-coders/vgstation13/pull/19559 but it'll be babby tier if it does.

:cl:
 * bugfix: There's now a message when you begin to place someone else into an inflatable shelter.
 * rscadd: You can now click-drag yourself and others into inflatable shelters.

### CODE REVIEW

File: code/game/objects/structures/inflatable.dm [hunk 1]
```diff
@@ -439,3 +439,41 @@
 			update_icon()
 			to_chat(user,"<span class='notice'>You climb free of the shelter.</span>")
 
+/obj/structure/inflatable/shelter/MouseDropTo(atom/movable/O as mob|obj, mob/user as mob) //copy pasted from cryo code
```
Comment (Kurfursten) [inline code/game/objects/structures/inflatable.dm]: you can remove "as X"

File: code/game/objects/structures/inflatable.dm [hunk 2]
```diff
@@ -439,3 +439,41 @@
...
+	if(O.locked_to)
+		return
+	else if(O.anchored)
+		return
+	if(issilicon(O)) //robutts dont fit
+		return
+	if(!ishigherbeing(user) && !isrobot(user)) //No ghosts or mice putting people into the sleeper
+		return
+	if(isrobot(user))
+		var/mob/living/silicon/robot/robit = usr
```
Comment (Kurfursten) [inline code/game/objects/structures/inflatable.dm]: user instead of usr

File: code/game/objects/structures/inflatable.dm [hunk 3]
```diff
@@ -439,3 +439,41 @@
...
 			to_chat(user,"<span class='notice'>You climb free of the shelter.</span>")
 
+/obj/structure/inflatable/shelter/MouseDropTo(atom/movable/O as mob|obj, mob/user as mob) //copy pasted from cryo code
+	if(!ismob(O))
+		return
+	if(O.loc == user || !isturf(O.loc) || !isturf(user.loc) || !user.Adjacent(O)) //no you can't pull things out of your ass
+		return
+	if(user.incapacitated() || user.lying) //are you cuffed, dying, lying, stunned or other
+		return
+	if(!Adjacent(user) || !user.Adjacent(src) || user.contents.Find(src)) // is the mob too far away from you, or are you too far away from the source
```
Comment (Kurfursten) [inline code/game/objects/structures/inflatable.dm]: user.contents.Find(src) shouldn't apply so you can remove that

File: code/game/objects/structures/inflatable.dm [hunk 4]
```diff
@@ -439,3 +439,41 @@
 			update_icon()
 			to_chat(user,"<span class='notice'>You climb free of the shelter.</span>")
 
+/obj/structure/inflatable/shelter/MouseDropTo(atom/movable/O as mob|obj, mob/user as mob) //copy pasted from cryo code
+	if(!ismob(O))
```
Comment (Kurfursten) [inline code/game/objects/structures/inflatable.dm]: if you're going to check to make sure O is a living down at 464 this is redundant

File: code/game/objects/structures/inflatable.dm (hunk 3 above)
Comment (ProfanedBane) [reply code/game/objects/structures/inflatable.dm]: In theory I think it's for if you have a mob in your hands (ie, Ian, the pets etc).

**Comment by ProfanedBane** [comment]
APC destroyed mission accomplished.

**Comment by ProfanedBane** [comment]
@jknpj Conflicts are kill.

**Pull request merged**